.travis.yml

```diff
@@ -55,11 +50,6 @@ matrix:
       TEST_GEM: chef-provisioning-aws
     script: tasks/bin/run_external_test $TEST_GEM rake spec
     rvm: 2.2
-# requires vagrant
```

Comment:
yard docs for now.  since we just release 12.9.x it'll be a little while before 12.10.x ships.  i also want to compat_resource those APIs and port chef-rewind over to use them instead of it applying its own monkeypatches to the resource collection.   and we don't even have compat_resource for 12.9.x and i'm waiting for john to help out with getting the recursive notifications resource_collection patches merged in there before tackling the 12.10.x compat_resource version...   then i'd like to doc them and blog about using them, and maybe even yap a bit about them all at chefconf...